### ISSUE TITLE
iOS - Cannot set title color and size

### ISSUE THREAD

**Issue by kheraprabhat** [wix/react-native-navigation#6158]
### Issue Description

largeTitle - color property not working
title - color property not working properly and style 

[Video Link](https://i.imgur.com/I8k7gbl.mp4)
Video Link: https://i.imgur.com/I8k7gbl.mp4

### Steps to Reproduce / Code Snippets / Screenshots
My default options and stack options object is below. 

```
{
   "layout":{
      "orientation":[
         "portrait"
      ],
      "backgroundColor":"#f6f7f6"
   },
   "statusBar":{
      "backgroundColor":"#f6f7f6"
   },
   "topBar":{
      "barStyle":"default",
      "largeTitle":{
         "visible":true,
         "color":"#f6f7f6"
      },
      "visible":true,
      "drawBehind":true,
      "animate":true,
      "transparent":false,
      "translucent":false,
      "title":{
         "fontSize":22,
         "fontWeight":"regular",
         "color":"#f6f7f6"
      },
      "background":{
         "color":"#1864C0"
      },
      "backButton":{
         "color":"#f6f7f6"
      }
   }
}
```

---
### Environment
* React Native Navigation version: 6.4.0
* React Native version: 0.61.5
* Platform(s) (iOS, Android, or both?): iOS
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator and Device, OS: 13.4.1 Debug

**Comment by yanniz0r** [comment]
I encountered the same issue. I think it's somehow related to the iOS 13.x as I was not able reproduce it in iOS 12.x. Furthermore, it seems like the color of the large title has some kind of relation to the dark mode of iOS. My large title was white for dark mode and black for light mode.

I was able to set it fixed to white by adding the following to my Info.plist:
```
<key>UIUserInterfaceStyle</key>
<string>Dark</string>
```
